src/test/resources/features/dcu/InitialDraft.feature

```diff
@@ -13,15 +13,19 @@ Feature: Initial Draft
     Then an error message appears instructing me to add rejection reasons
 
```

Comment:
For below scenario, does DTEN not have telephone option?

Reply:
DTEN don't have an option to select a response channel at Initial draft